api/devices/schema.py

```diff
@@ -94,7 +94,9 @@ class Query(graphene.ObjectType):
     """
     all_devices = graphene.List(
         Devices,
-        description="Query that returns a list of all devices")
+        device_labels=graphene.String(),
+        description="Query that returns a list of all devices accepts the argument\
```

Comment:
The description should also change because if you filter by labels the query will not  all labels

Reply:
fixed @mnswaleh